cmake/findBinpkgs/FindIconv.cmake

```diff
@@ -3,8 +3,8 @@
 # ICONV_LIBRARIES - Link these to use Iconv 
 
 find_path(ICONV_INCLUDE_DIR iconv.h) 
-find_library(ICONV_LIBRARIES NAMES iconv libiconv libiconv-2 c)
+find_library(ICONV_LIBRARIES NAMES iconv libiconv libiconv-2)
```

Comment:
I wonder why he did that... Maybe it's the right way on Android but I'm not sure we want that on every platform. I'm pretty sure this is useful for instance on Debian because there is not iconv package.